### PR TITLE
update JavaFX dependencies to build with JDK11

### PR DESCRIPTION
**Pull request by ianturton**
Fix dependencies and generally update them.

### CODE REVIEW

File: modules/unsupported/javafx/README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ A JavaFX parent subclass containing a Canvas. A map from a WMS-Server is drawn t
 
 In development.
 
+Note: You need to [download OpenJDK](https://wiki.openjdk.java.net/display/OpenJFX/Download) to use this module.
```
Comment (aaime) [inline modules/unsupported/javafx/README.md]: Why so? Should work with Oracle too given the dependencies in the pom. Afaik it was not working with OpenJDK, not even before (OpenJDK did not package JavaFX)

Comment (aaime) [inline modules/unsupported/javafx/README.md]: Why so? Should work with Oracle too given the dependencies in the pom. Afaik it was not working with OpenJDK, not even before (OpenJDK did not package JavaFX)

Comment (ianturton) [reply modules/unsupported/javafx/README.md]: Well I had to download it, but I guess it should work with Oracle if they still package it. 
I can delete that line if you think it is confusing.

Comment (ianturton) [reply modules/unsupported/javafx/README.md]: Well I had to download it, but I guess it should work with Oracle if they still package it. 
I can delete that line if you think it is confusing.

Comment (ianturton) [reply modules/unsupported/javafx/README.md]: Well I had to download it, but I guess it should work with Oracle if they still package it. 
I can delete that line if you think it is confusing.

**Comment by mipastgt** [comment]
Did you have a look at this link? https://openjfx.io/openjfx-docs/#maven Starting with JavaFX11 you don't have to download anything anymore. You can just add the corresponding Maven/Gradle dependencies.

**Comment by aaime** [comment]
@mipastgt  That's what has been done indeed
@ianturton Looks good to me

**Pull request merged**